REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
Oh, one other problem with GSL.jl : it doesn't compile on Edison. That
didn't stop me before because we just need GSL.jl for tests, and I'd test
it locally before running it there. But, still a pain.

On Thu, Nov 19, 2015 at 3:41 PM Ryan <notifications@github.com> wrote:

> In REQUIRE
> <https://github.com/jeff-regier/Celeste.jl/pull/89#discussion_r45418041>:
>
> > @@ -4,7 +4,7 @@ Distributions
> >  Docile
> >  DualNumbers
> >  FITSIO
> > -ForwardDiff 0.0.3 0.1
> > +ForwardDiff # Currently only required for testing.
>
> I'm looking into it. Superficially it doesn't look as easy as using
> DualNumbers. I think they didn't intend for anyone to interact with their
> data types except through the ForwardDiff API, but we have to in order to
> use our special data structures. It would be really nice if we could,
> though, for many reasons.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/89/files#r45418041>.
>